azuki-core/src/main/kotlin/com/anaplan/engineering/azuki/core/parser/ScenarioParser.kt

```diff
@@ -2,11 +2,47 @@ package com.anaplan.engineering.azuki.core.parser
 
 import com.anaplan.engineering.azuki.core.scenario.BuildableScenario
 
-interface ScenarioParser<S: BuildableScenario<*>> {
+interface ScenarioParser<S : BuildableScenario<*>> {
 
     fun parse(
-        scenarioString: String,
-        requiredImports: String
+        scenarioString: String, requiredImports: String
     ): S
```

Comment:
❓ would it be better to create a new interface for exposing the newer API instead of jury-rigging the old one to have the new API fall back to it?  I'm not sure what the usual best practice is for evolving APIs here, and there is definitely a lot of downstream that consumes this interface.